types/react-image-magnify/index.d.ts

```diff
@@ -0,0 +1,84 @@
+// Type definitions for react-image-magnify 2.7
+// Project: https://github.com/ethanselzer/react-image-magnify
+// Definitions by: Muhammet Öztürk <https://github.com/hlthi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from 'react';
+
+export default class ReactImageMagnify extends React.Component<ReactImageMagnifyProps, any> {
+    /*
```

Comment:
Is this comment documentation or just leftover? Either way it's probably better to get rid of it.